### DISCUSSION TITLE
RegEx Help

### DISCUSSION THREAD

**Discussion by tsuliwaensis** [openstyles/stylus#1476] [Q&A] [answered]
How would I craft a RegEx that will make a style apply only to the inner pages of a website? For example, I want the style to apply to `https://www.example.com/this-is-a-page?url=1` **but not** `https://www.example.com/`. So anything after the last `/`

**Comment by tophf** [comment, marked as answer]
`^https?://www\.example\.com/[^?#].*` this one also excludes parameters for the main site after ? and #

**Reply by tsuliwaensis** [reply]
This is perfect. Thank you!

**Comment by decembre** [comment]
I have too a problem with Regexp in namespae.

This one don't only match "quora.com" but all sites:
@-moz-document domain("www.quora.com"), url-prefix("https://quora.com"), regexp('^https:\\/\\/(?!quora.com*)\\S+\\/*')

It's from a request i do here:
<a href="https://greasyfork.org/fr/discussions/development/149680-help-need-regexp-for-quora-com-for-namespace-regexp-xxx#comment-329506">Help Need regexp for Quora.com (for namespace regexp'&quot;XXX&quot;)</a>

And the regexp test is here:
https://regex101.com/r/ZKnDnR/1

Can you correct it ?

**Reply by tophf** [reply]
Assuming you want to run on all \*.quora.com sites including quora.com, all you need is `@-moz-document domain(quora.com) {`

**Reply by decembre** [reply]
Haaa ok, i don't realize how it was simple:

@-moz-document domain(quora.com) { 
is perfect
